src/config/mod.rs

```diff
@@ -173,6 +173,10 @@ pub fn feature_flag_balances_rate_implementation() -> bool {
     env_with_default("FEATURE_FLAG_BALANCES_RATE_IMPLEMENTATION", false)
 }
 
+pub fn is_preview_endpoint_enabled() -> bool {
+    env_with_default("FEATURE_PREVIEW_ENDPOINT", false)
+}
+
```

Comment:
One question, should we add `FEATURE_PREVIEW_ENDPOINT` to `.env.sample`? what do you think?

Reply:
Added in [fe778a4](https://github.com/safe-global/safe-client-gateway/pull/924/commits/fe778a476fa653ec794139060cc20ccf6d22ce4a)